ethereum/statetransition/src/test-support/java/tech/pegasys/artemis/statetransition/AttestationGenerator.java

```diff
@@ -46,6 +48,26 @@ public AttestationGenerator(final List<BLSKeyPair> validatorKeys) {
     this.validatorKeys = validatorKeys;
   }
 
+  public static Attestation aggregateAttestation(int numAttesters) {
+    Attestation attestation = DataStructureUtil.randomAttestation(1);
+    withNewAttesterBits(attestation, numAttesters);
+    return attestation;
+  }
+
+  public static Attestation withNewAttesterBits(Attestation attestation, int numNewAttesters) {
+    Bitlist newBitlist = new Bitlist(attestation.getAggregation_bits());
+    int numBitsSet = 0;
+    while (numBitsSet < numNewAttesters) {
```

Comment:
(optional) - you could create a list of indices that are 0, sort randomly, and then set the first `numNewAttesters` indices.  That allows you to easily avoid an infinite loop if too many bits are already set.

Reply:
done